package.json

```diff
@@ -77,7 +77,7 @@
     "@changesets/parse": "^0.3.16",
     "@changesets/pre": "^1.0.14",
     "@changesets/read": "^0.5.9",
-    "@gitbeaker/node": "^35.7.0",
+    "@gitbeaker/rest": "^39.23.0",
```

Comment:
Thanks for your work first! It's realy a good start. Although this change should be marked as minor change instead of patch. It's a feature change instead of bug fix.

Reply:
Updated!